src/System.Reflection.TypeExtensions/tests/project.json

```diff
@@ -1,9 +1,9 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23901",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23903",
```

Comment:
For some reason updating just the packaging did not work. At runtime, it was still picking up System.Reflection that did not have the new TypeForwarders like BindingFlags (although the package lib has it). I tried digging in but finally gave up and used project reference as well.